docs/roadmap.md

```diff
@@ -58,7 +58,7 @@ clustered database or key-value store. We will target such workloads for our
   - Status: in progress
 3. Scale to 30-50 pods (1-2 containers each) per node (#4188)
   - Status:
-4. Scheduling throughput: 99% of scheduling decisions made in less than 1s on 100 node, 3000 pod cluster; linear time to number of nodes and pods (#3954)
+4. Scheduling throughput: 99% of end-to-end pod startup time with prepulled images is less than 5s on 100 node, 3000 pod cluster; linear time to number of nodes and pods (#3952, #3954)
```

Comment:
Why is this the scheduling throughput if we include kubelet pulling the image and starting the containers?